vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/Holder.kt

```diff
@@ -156,7 +156,23 @@ interface Holder {
         inputDescriptors: Collection<InputDescriptor>,
         fallbackFormatHolder: FormatHolder? = null,
         pathAuthorizationValidator: PathAuthorizationValidator? = null,
-    ): KmmResult<Map<InputDescriptor, Map<SubjectCredentialStore.StoreEntry, Map<ConstraintField, NodeList>>>>
+    ): KmmResult<Map<String, Map<SubjectCredentialStore.StoreEntry, Map<ConstraintField, NodeList>>>>
```

Comment:
Maybe a typealias for the long `Map` construct would help reading the code?

Reply:
fixed in f2a300dabe9855916935f35daf1408b409f71cc8